index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="/Styles/styles.css" />
```

Comment:
- [ ] In order to ensure that your website CSS loads properly in the production environment across browsers, I will recommend you add a dot(.) in front of tour relative path e.g. `<link rel="stylesheet" href="./Styles/styles.css" />`

PLease do this as your CSS file is not loading properly on the browser 👍🏽

Reply:
okay let me do it sir